### ISSUE TITLE
♻️ `checkIntegrity` no longer needed

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#9771]
## ♻️ Debt

The `check-integrity` script is no longer needed after the installation of pnpm.

## 🙋‍♀️ Proposed Solution

Remove any references to `checkIntegrity` including `packages/tooling/src/checkIntegrity.ts` file and the `check-integrity` script.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the debt was addressed and expected functionality has not been affected.

- [ ] no references to `checkIntegrity` in codebase

## 🛑 Blockers

Issues which must be completed before this one.

```[tasklist]
### Blocked By
- [ ] #5338
```
